Commands/notify.py

```diff
@@ -308,8 +310,8 @@ async def add_list(self, ctx: commands.Context, list_name: str):
                 custom_emoji = False
 
             # Error if emoji is being used already on this server
-            for list_name, list_data in guild_data.notification_lists.items():
-                if reaction_emoji == list_data["emoji"]:
+            for name, data in guild_data.notification_lists.items():
```

Comment:
since the name is not used in this loop, you can use `for data in guild_data.notification_lists.values():`